File: 2014/J1.java
import java.util.Scanner;

public class Main {
	
	public static void main(String args[]) {
		Scanner scanner = new Scanner(System.in);
		
    // get input
		int a1 = scanner.nextInt();
		int a2 = scanner.nextInt();
		int a3 = scanner.nextInt();
		
    //check conditions
		if ((a1 + a2 + a3) != 180) System.out.println("Error");
		else if (a1 == a2 && a2 == a3) System.out.println("Equilateral");
		else if (a1 == a2| a2 == a3 | a1 == a3) System.out.println("Isosceles");
		else if (a1 != a2 && a1 != a3 && a2 != a3) System.out.println("Scalene");
		
		scanner.close();
	}
}

File: 2014/python/J1.py
# Alec Dewulf
# "Triangle Times" J1 2014

a1 = int(input())
a2 = int(input())
a3 = int(input())

if a1 + a2 + a3 == 180:
    if a1 == 60 and a2 == 60 and a3 == 60:
        print("Equilateral")
    elif a1 == a2 or a1 == a3 or a2 == a3:
        print("Isosceles")
    elif a1 != a2 and a1 != a3 and a2 != a3:
        print("Scalene")
else:
    print("Error")
